index.js

```diff
@@ -12,30 +12,19 @@
  * the License.
  */
 
-const merge = require('lodash/merge');
-const chalk = require('chalk');
-const buildToMatchSchema = require('./matchers/toMatchSchema');
-const buildToBeValidSchema = require('./matchers/toBeValidSchema');
+const toMatchSchema = require('./matchers/toMatchSchema');
+const toBeValidSchema = require('./matchers/toBeValidSchema');
 
-function matchersWithOptions(userOptions = {}) {
-  const defaultOptions = {
-    allErrors: true,
-  };
+const { expect } = global;
 
-  const options = merge(defaultOptions, userOptions);
+const matchers = {
+  toMatchSchema,
+  toBeValidSchema,
+};
 
-  return {
-    toMatchSchema: buildToMatchSchema(options),
-    toBeValidSchema: buildToBeValidSchema(options),
-  };
+/* istanbul ignore next */
```

Comment:
I've written a positive test case for this but I can't think of a way of writing a negative test case – open to ideas.